.github/workflows/verify-browserstack.yml

```
@@ -1,11 +1,23 @@
 name: Verify Browserstack
 
-on: pull_request
+on:
+  # Run the workflow when commits are pushed to the main branch
+  push:
+    branches:
+      - master
+  # Also run when a PR is opened
+  pull_request:
```

```suggestion
on: pull_request
```

if they run for our own pull request then it's fine... on master this would only add noise if they are flaky

---

My reason was that if you accept a PR from someone else, you never run these tests at all. If that is okay, then we can skip running on `master`.